editor/src/clj/editor/render_pb.clj

```diff
@@ -118,7 +118,8 @@
 
 (defn- build-errors
   [_node-id script named-render-resources]
-  (when-let [errors (->> (into [(validation/prop-error :fatal _node-id :script validation/prop-resource-missing? script "Script")]
+  (when-let [errors (->> (into [(or (validation/prop-error :fatal _node-id :script validation/prop-resource-missing? script "Script")
+                                    (validation/prop-error :fatal _node-id :script validation/prop-resource-ext? script "render_script" "Script"))]
```

Comment:
There is no property for that field, only a form view. But it's rarely used thing, so I thing preventing it from building is good enough.